crates/rolldown_loader_utils/src/json_to_esm.rs

```diff
@@ -3,9 +3,13 @@ use rolldown_utils::ecma_script::is_validate_assignee_identifier_name;
 use serde_json::Value;
 // TODO: handling https://github.com/tc39/proposal-json-superset
 
+fn trim_bom(raw: &str) -> &str {
+  raw.strip_prefix("\u{FEFF}").unwrap_or(raw)
```

Comment:
Isn't this just https://doc.rust-lang.org/std/primitive.str.html#method.trim_start_matches